src/hub.js

```diff
@@ -441,6 +441,10 @@ async function updateEnvironmentForHub(hub, entryManager) {
         );
 
         sceneEl.emit("leaving_loading_environment");
+        if (environmentEl.components["gltf-model-plus"].data.src === sceneUrl) {
+          console.warn("Updating environment to the same url.");
+          environmentEl.setAttribute("gltf-model-plus", { src: "" });
+        }
```

Comment:
This won't happen in "normal conditions" -- maybe solve this by adding a qs parameter to `loadingEnvironment`. Also we should just audit the fader code/usage because it's suspicious.

Won't merge this.